waku/waku_store/protocol.nim

```diff
@@ -55,6 +63,18 @@ proc initProtocolHandler(ws: WakuStore) =
       # TODO: Return (BAD_REQUEST, cause: "empty query")
       return
 
+    if ws.requestRateLimiter.isSome() and not ws.requestRateLimiter.get().tryConsume(1):
+      trace "store query request rejected due rate limit exceeded",
+        peerId = $conn.peerId, requestId = reqRpc.requestId
+      let error = HistoryError(kind: HistoryErrorKind.TOO_MANY_REQUESTS).toRPC()
+      let response = HistoryResponseRPC(error: error)
+      let rpc = HistoryRPC(requestId: reqRpc.requestId, response: some(response))
+      await conn.writeLp(rpc.encode().buffer)
+      waku_service_requests_rejected.inc(labelValues = ["Lightpush"])
+      return
+
+    waku_service_requests.inc(labelValues = ["Lightpush"])
```

Comment:
shouldn't be "Store" instead of "Lightpush"?

Reply:
Great catch!!!! Thank you.